### DISCUSSION TITLE
about-me

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#439] [Announcements]
# about-me

介绍页 本人毕业于一所边缘的本科院校，专业是软件工程，刚进入这个专业的时候，我连这个专业是做什么的都不是很清楚， 甚至有同学告诉我说，学校教的东西都不用学，只要拿到毕业证，学习一门编程语言就可以随便找工作，我当时甚至一度以为这是真的... 后来到了大二暑假的时候，那时候我还在家乡兼职暑假工，平时在家也不知道干什么，就开始自己学习，也不知道学什么，毫无方向， 在我一通乱学之下，我巧然打开了新世界的大门。 当时我是在阿里云的开发者社区中发现的学习路线图，然后就开始学习了，首先学的是前端的一些知识，像html,css,js等，为我之后进入java做一些铺垫，毕竟前端的知识还是有必要学习一下的。 大二开学，我就开始正式学习java了，直到大三结束，我才把java、mysql、spring、spirngmvc、springboot、myabits等学习完毕，在没有人帮助的情况下， 每次遇到bug都是自己解决，说实话，这真的很不容易！

http://localhost:8080/about-me.html

**Comment by MyRedScarf** [comment]
泪目
